warehouse/models/staging/gtfs/_src_gtfs_rt_external_tables.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: external_gtfs_rt
     description: Hive-partitioned external tables reading GTFS RT data and validation errors from GCS.
-    database: cal-itp-data-infra
+    database: "{{ var('source_database', 'cal-itp-data-infra') }}"
```

Comment:
Do you mean to be committing this here? If so, do you want to make this more systematic: actually define the var in the profile, define for other sources?

Reply:
I was conflicted because we don't always have a test version of data, and I imagine the bulk of sources work is done on relatively few sources.

I'll go ahead and change it, though; that's reasonable.